<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountRecovery</class>
 <widget class="QMainWindow" name="AccountRecovery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="usernameDisplay_LineEdit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>260</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="verificationCode_LineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recovery_PushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Recovery</string>
    </property>
   </widget>
   <widget class="QLabel" name="usernameDisplay_Lable">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>260</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnToMainWindow_PushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>370</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailInput">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QPushButton" name="verificationCode_PushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Verify</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwordDisplay_Lable">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>290</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLabel" name="emailContainAtSign_Lable">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Emails should contain @</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordDisplay_LineEdit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>290</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="verificationCode_Lable">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>A verification code has been send to your email.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>emailInput</tabstop>
  <tabstop>recovery_PushButton</tabstop>
  <tabstop>verificationCode_LineEdit</tabstop>
  <tabstop>verificationCode_PushButton</tabstop>
  <tabstop>usernameDisplay_LineEdit</tabstop>
  <tabstop>passwordDisplay_LineEdit</tabstop>
  <tabstop>returnToMainWindow_PushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
